docs/source/diagrams/ReflectometryReductionOne_LowLvl-v1_wkflw.dot

```diff
@@ -55,17 +66,28 @@ $algorithm_style
   rebin           [label="Rebin"]
   scale           [label="Scale"]
   calcResolution  [label="CalculateResolution"]
+  convertDet      [label="Convert Units\n(AlignBins = True)"]
+  convertMon      [label="Convert Units"]
+  groupDet        [label="Group Detectors"]
+  cropDetWS       [label="Crop Detector\nWorkspace"]
+  rebinDet        [label="Rebin Detectors"]
```

Comment:
We should use the names of the algorithms above. For instance, `Rebin Detectors` should be just `Rebin` (there's no algorithm called like that. I don't think it is necessary here to explicitly say we are rebinning the detector workspace, as we already have two separate paths for detectors and monitors).

The same for the rest of the algorithms:

- `Convert Units\n(AlignBins = True)` -> `ConvertUnits\n(AlignBins=True)` (In this case I think it's OK to leave the `AlignBins = True` there).
- `Convert Units` -> `ConvertUnits`
- `Group Detectors` -> `GroupDetectors`
- `CropDetector\nWorkspace` -> `CropWorkspace`